extensions/amp-analytics/amp-analytics.md

```diff
@@ -319,13 +319,25 @@ The `triggers` attribute describes when an analytics request should be sent. It
   - `timerSpec` (required when `on` is set to `timer`) This configuration is used on conjunction with the `timer` trigger. Please see below for details.
 
 #### Page visible trigger (`"on": "visible"`)
-Use this configuration to fire a request when the page becomes visible. No further configuration is required.
+Use this configuration to fire a request when the page becomes visible. The firing of this trigger can be configured using `visibilitySpec`. Configuration properties supported in `visibilitySpec` are:
+  - `selector` This property can be used to specify the element to which all the `visibilitySpec` conditions apply.
+  - `continuousTimeMin` and `continuousTimeMax` These conditions keep track of the amount of time (any part of) an element has been in the viewport and fire when the time is between the minimum and maximum configured time. The time is expressed in milliseconds.
+  - `totalTimeMin` and `totalTimeMax` These conditions keep track of total time for which any part of an element is on screen and fire when the time is between minimum and maximum. The time is expressed in milliseconds.
+  - `visiblePercentageMin` and `visiblePercentageMax` These conditions keep track of the amount of element visible in vieweport at any given time. They get triggered when the percentage of element visible is between min and max (if specified). Valid values are 0-100 (lower bound is not inclusive while the upper bound is).
+
+In addition to the conditions above, `visibilitySpec` also enables certain variables which are documented [here](./analytics-vars.md#visibility-variables).
```

Comment:
Assuming the statement I write here is true, should have something like: "If multiple properties are specified, they must all be true in order for a request to fire."